src/qz/printer/action/PrintHTML.java

```diff
@@ -71,10 +73,13 @@ public void parseData(JSONArray printData, PrintOptions options) throws JSONExce
 
             for(int i = 0; i < printData.length(); i++) {
                 JSONObject data = printData.getJSONObject(i);
-                String source = data.getString("data");
 
                 PrintingUtilities.Flavor flavor = PrintingUtilities.Flavor.valueOf(data.optString("flavor", "FILE").toUpperCase(Locale.ENGLISH));
 
+                String source = flavor == PrintingUtilities.Flavor.BASE64 ?
```

Comment:
This should probably be an if statement like the other pixel processors so it'd be easier later on if we need to add other flavor handlers.

Reply:
They have been in the subsequent PR.

Reply:
Done.